src/Faker/Provider/id_ID/Color.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Faker\Provider\id_ID;
+
+class Color extends \Faker\Provider\Color
+{
+    
+    /**
+     * @link https://id.wikipedia.org/wiki/Daftar_warna
+     **/
+    protected static $safeColorNames = array(
+        'abu-abu', 'biru', 'biru dongker', 'biru laut', 'coklat', 'emas', 'hijau', 'hitam', 'kuning',
+        'magenta', 'mawar', 'merah', 'merah marun', 'merah jambu', 'nila', 'jingga', 'perak', 'putih', 'ungu',
+        'cokelat',
+    );
+    
+    /**
+     * @link http://listeningsection.blogspot.com/2017/08/100-nama-warna-dalam-bahasa-inggris.html
```

Comment:
the content on this site is copyrighted. Please find a datasource compatible with Faker's MIT license

Reply:
sorry for my mistake, now i fix it use data from wikipedia, but some data i change with manual translate